Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : Date {get}
```

Comment:
Can we also change `time` and `endTime` to a `DateInterval`?

Reply:
Cool - this takes out the duration property on `HackathonEvent` too :)